index.html

```diff
@@ -1584,7 +1619,17 @@ <h2>
           <a>PaymentDetailsBase</a> dictionary, the following members are part
           of the <a>PaymentDetailsInit</a> dictionary:
         </p>
-        <dl>
+        <dl data-sort="">
+          <dt>
+            <dfn>lang</dfn> member
+          </dt>
+          <dd>
+            A [[!BCP47]] language tag representing the preferred language of
```

Comment:
s/preferred/merchant's preferred/ (what the merchant prefers might not be what the buyer prefers)